MessengerOnFileForLinux/CMakeLists.txt

```diff
@@ -119,10 +119,13 @@ target_link_libraries(messenger_binary_UT ${CURSES_LIBRARIES}
 
 ################################################################################
 #####                      SET COMPILATION ORDER                           #####
-##### 1.messenger_binary                                                   #####
-##### 2.messenger_binary_UT                                                #####
+##### 1.source_objects                                                     #####
+##### 2.messenger_binary                                                   #####
```

Comment:
dodać (source objects in use), albo coś w tym stylu, to samo dla *_UT

Reply:
done